src/components/questions/FormAnswer.jsx

```diff
@@ -8,6 +8,9 @@ const AnswerForm = (props) => {
   const {info} = useProductContext();
   const productName = info.name;
 
+  const {questions} = useProductContext();
```

Comment:
if you get the  {questions} from context,  you don't need to pass it all the way down from QuestionList.jsx.